tailor_image/create_image.py

```diff
@@ -28,17 +27,18 @@ def create_image(name: str, distribution: str, apt_repo: str, release_track: str
     :param flavour: Bundle flavour to install on the images
     :param organization: Name of the organization
     :param docker_registry: URL for the docker registry to use to push images from/to
-    :param rosdistro_path: Path for the rosdistro configuration files
     :param publish: Whether to publish the images
     """
 
     # Read configuration files
+    rosdistro_path = pathlib.Path('/rosdistro')
```

Comment:
is there a reason you don't want to read it as an arg? I find that helps when running `tailor-distro` CI commands locally for development, but I haven't worked with `tailor-image`.

(https://github.com/locusrobotics/tailor-distro#development)

Reply:
I've removed it since I was hardcoding it anyways, but I think it makes sense if you want to run the script locally for testing.